src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx

```diff
@@ -37,15 +37,24 @@ export default {
     },
 };
 
-export const AddTemplateDropdownMenu: StoryObj<typeof MetadataSidebarRedesign> = {};
+export const AddTemplateDropdownMenu: StoryObj<typeof MetadataSidebarRedesign> = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        await canvas.findAllByRole('heading', { name: 'Metadata' });
+
+        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' });
```

Comment:
When I deleted the line above in my stories, they sometimes failed to find the button, so I added an additional timeout to ensure we wait for the button to render
`const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' }, { timeout: 5000 });
`